.github/workflows/release-docker.yml

```diff
@@ -14,7 +14,10 @@ jobs:
 
     steps:
       - name: Login to Docker Hub
-        run: echo "${{ secrets.DOCKER_PASSWORD }}" | docker login -u "${{ secrets.DOCKER_USERNAME }}" --password-stdin
+        uses: docker/login-action@f4ef78c080cd8ba55a85445d5b36e214a81df20a # v2.1.0
+        with:
+          username: ${{ secrets.DOCKER_USERNAME }}
+          password: ${{ secrets.DOCKER_PASSWORD }}
```

Comment:
How can we test this to be sure it's gonna work?  It's really tricky because this only triggers on creation of a github release, and that has already happened with v0.5.9.  I suppose we'll get the first test with v0.6-beta... which should be pretty soon anyway.

Reply:
I test the workflow changes like this: 

1. Create new GH repo (not a fork)
2. Configure docker hub secrets (to my own Docker account)
3. Push dcrdex master to it
4. Push work branch
5. Change `IMAGE_NAME` in GH action to point to my DH
6. Create release
7. If there's a problem GOTO #4

It's a major PITA but everything is covered.